sql/views.sql

```diff
@@ -55,12 +55,14 @@ CREATE OR REPLACE VIEW timescaledb_information.job_stats as
                    WHEN js.last_run_success = 'f' THEN 'Failed'
               END
     END as last_run_status,
-    CASE WHEN js.last_finish < '4714-11-24 00:00:00+00 BC' THEN 'Running'
+    CASE WHEN j.scheduled = false THEN 'Paused'
+         WHEN js.last_finish < '4714-11-24 00:00:00+00 BC' THEN 'Running'
```

Comment:
i think you should do outer join against pg_stat_activity here to get running status

Reply:
use "Not scheduled" instead of "Paused"?

Reply:
@svenklemm We should be able to infer the status based on ( scheduled= true and last_finish ) without resorting to an outer join.

Reply:
"Not scheduled" seems like a poor semantic fit for `job_status`.  Maybe "Stopped" makes more sense here than "Paused"?